docs/quickstart.rst

```diff
@@ -360,23 +374,22 @@ In this case, the symbol must further be imported from ``__externals__`` in the
     ):
         from __externals__ import USE_ALPHA
 
-        if __INLINED(USE_ALPHA):
-            with computation(PARALLEL), interval(...):
-                result = field_a[0, 0, 0] - (1 - alpha) * (
+        with computation(PARALLEL), interval(...):
+            if __INLINED(USE_ALPHA):
+                result = field_a[0, 0, 0] - (1. - alpha) * (
                     field_b[0, 0, 0] - weight * field_c[0, 0, 0]
                 )
-        else:
-            with computation(PARALLEL), interval(...):
+            else:
```

Comment:
This change was from @ofuhrer and I believe it was suggested because in this example the computation regions are the same in both the 'if' and the 'else' case, so in practice it would make more sense to do the if inside of the 'with computation'. But, I also agree, having an example that demonstrates a conditional can be done outside of the computation is useful. Given the syntax is going to change soon anyways, I am ok with reverting it. Or alternatively we could make an example with a slightly different computation in the else case?

Reply:
Thanks for the explanation, I see the point now. I think the best solution would be to make an example with a slightly different computation in the else case as you suggest. Would you agree? Would you have time to quickly add it to this PR?

Reply:
Done.